exporter/dorisexporter/README.md

```diff
@@ -0,0 +1,72 @@
+# Apache Doris Exporter
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: traces, metrics, logs   |
+| Distributions | [] |
```

Comment:
Do you need to fill in `[contrib]` here?

Reply:
CONTRIBUTING.md says that I need to add it in the last PR.